### PR TITLE
docs(themes): add docs for schemas

### PR DESCRIPTION
**Pull request by simeonoff**

### CODE REVIEW

File: en/components/avatar.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Analogically, the avatar can display an icon via the [`icon`]({environment:angul
 ### API References
 <div class="divider--half"></div>
 
-* [IgxAvatarComponent]({environment:angularApiUrl}/classes/igxavatarcomponent.html)
+* [IgxAvatarComponent]({environment:sassApiUrl}/classes/igxavatarcomponent.html)
```
Comment (HristoP96) [inline en/components/avatar.md]: Shouldn't this line reference the [IgxAvatar typedoc docs](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxavatarcomponent.html)?

Comment (simeonoff) [reply en/components/avatar.md]: You are absolutely right. Fixed with the latest commit.

File: en/components/themes/global-theme.md [hunk 2]
```diff
@@ -61,13 +62,41 @@ $unnecessary: (igx-avatar, igx-badge);
...
 <div class="divider--half"></div>
 
 The value of `$igx-legacy-support` is quite important as it determines how component themes will work. When its value is set to `true`, individual component style rules will have their values set at build time to the hard values defined in their theme. If you set the value of `$igx-legacy-support` to `false`, however, style rules will look for values from CSS variables defined at the `:root` scope, or the nearest block scope.
 
 The general rule of thumb regarding what the value of `$legacy-support` should be is dictated by whether you will be including support for Internet Explorer 11 or not. If you want to include support for IE11 set the `$legacy-support` value to `true` (default), otherwise setting its value to `false` will force CSS variables for theming.
 
+### API Overview
+* [Global Theme]({environment:sassApiUrl}/index.html#mixin-igx-theme)
+* [Light Theme]({environment:sassApiUrl}/index.html#mixin-igx-light-theme)
+* [Dark Theme]({environment:sassApiUrl}/index.html#mixin-igx-dark-theme)
```
Comment (HristoP96) [inline en/components/themes/global-theme.md]: Mixins `mixin-igx-light-theme` and `mixin-igx-dark-theme` are not included in the documentation.
The above defined Light Theme and Dark Theme links do not reference the required themes.

Comment (HristoP96) [reply en/components/themes/global-theme.md]: Ok

**Pull request merged**